### DISCUSSION TITLE
Error: Configuration file does not exis

### DISCUSSION THREAD

**Discussion by theguru06** [eclipse-edc/Connector#2611] [General]
While launching  dataspace-connector.jar usign command  
java -jar launchers/ids-connector/build/libs/dataspace-connector.jar
getting error:
 Configuration file does not exist: dataspaceconnector-configuration.properties. Ignoring.
INFO 2023-03-23T12:38:30.6165065 Initialized FS Configuration
SEVERE 2023-03-23T12:38:30.9304853 Error booting runtime: Vault file does not exist: dataspaceconnector-vault.properties

Anyone got this issue? Could someone please help me out how to run the [application?](![runEdc](https://user-images.githubusercontent.com/20825048/227129655-d380d099-2059-442d-9c87-b9cb0a3cedd9.png))
![runEdc](https://user-images.githubusercontent.com/20825048/227130016-2c66f3de-ffd2-47a0-b13e-feb44e6ebc3f.png)

**Comment by jimmarino** [comment]
Did you go through the onboarding documentation?
